tests/playwright/src/runner/podman-desktop-runner.ts

```diff
@@ -200,6 +203,30 @@ export class PodmanDesktopRunner {
       console.log(`Video file saved as: ${videoPath}`);
     }
     this._running = false;
+    await this.removeTracesOnFinished();
+  }
+
+  async removeTracesOnFinished(): Promise<void> {
+    const rawTracesPath = join(this._testOutput, 'traces', 'raw');
+
+    if (existsSync(rawTracesPath)) {
+      console.log(`Removing raw traces folder: ${rawTracesPath}`);
+      rmSync(rawTracesPath, { recursive: true, force: true, maxRetries: 5 });
+    }
+
+    if (this._testStatus && !process.env.KEEP_TRACES_ON_PASS && this._videoAndTraceName) {
```

Comment:
I would put second and third condition into subconditions like this:
```
if (testPassed) {
  if (KEEP_TRACES)
  if (KEEP_VIDEO)
}
```
or something like this.

Reply:
done.